packages/angular/src/generators/ngrx/__snapshots__/ngrx.spec.ts.snap

```diff
@@ -284,16 +283,15 @@ exports[`ngrx should add an empty root module when minimal and root are set to t
 import { StoreModule } from '@ngrx/store';
 import { EffectsModule } from '@ngrx/effects';
 import { StoreDevtoolsModule } from '@ngrx/store-devtools';
-import { environment } from '../environments/environment';
 import { StoreRouterConnectingModule } from '@ngrx/router-store';
      @NgModule({
        imports: [BrowserModule, RouterModule.forRoot([]), StoreModule.forRoot({}, {
-      metaReducers: !environment.production ? [] : [],
+      metaReducers: [],
       runtimeChecks: {
         strictActionImmutability: true,
         strictStateImmutability: true
       }
-    }), EffectsModule.forRoot([]), !environment.production ? StoreDevtoolsModule.instrument() : [], StoreRouterConnectingModule.forRoot()],
```

Comment:
Is this okay? For devtools to be instrumented in production?

Reply:
It most likely is not, no, but angular has no built-in `environment` object anymore coming from `environment` and `environment.prod.ts` files as these also no longer exist.

Reply:
Ok, so there's no solution from angular to handle this so I'll make sure we don't let them get into production